assets/constants.ts

```diff
@@ -5353,6 +5353,9 @@ export = {
   // Used to disable account data export until it is fully implemented.
   "ENABLE_ACCOUNT_EXPORT": false,
 
+  // Link to open when the Oppia avatar is clicked on any page.
+  "OPPIA_AVATAR_IMAGE_LINK": "https://www.who.int/emergencies/diseases/novel-coronavirus-2019/advice-for-public/when-and-how-to-use-masks",
```

Comment:
@aks681 This is more of a temporary thing. Could you please:

- Set it to null (we will update the link during specific deployments)
- For callers: If it is null, then nothing should happen; otherwise, if there's a link, we make the avatar clickable.

In particular, callers should handle the null case.

Reply:
Done